docs/guide/search.md

```diff
@@ -299,9 +299,27 @@ $searchResult = $searchService->findContent( $query );
 $content = $searchResult->searchHits[0];
 ```
 
-### Reindexing
+## Reindexing
 
-To (re)create the search engine index for configured search engines (per siteaccess repository), use the `php app/console ezplatform:reindex` command.
+To (re)create or refresh the search engine index for configured search engines (per siteaccess repository), use the `php app/console ezplatform:reindex` command.
+
+Some examples of common usage _(using 1.7.7/1.13 and up)_:
```

Comment:
I'd prefer leaving out `_(using 1.7.7/1.13 and up)_` -> We assume users update to the last patch version, and we just wouldn't merge this change to the 1.12 branch.

Reply:
I'm ok with that.

Note: 1.7.7 is not out yet, planned for end of january
Note2: This was actually added in 1.12.1 and up, so you may merge this to 1.12 branch if you want, but as it's a FTR and now superseded by 1.13 I would probably skip it :)

Reply:
Yes, we'll remove the 1.12 version from the doc output soonish anyway.